populate_db.py

```diff
@@ -0,0 +1,107 @@
+import pyphen, requests, json
+#pyphen.language_fallback('ipa_en')
+ipa_dic = pyphen.Pyphen(lang='ipa_en')
+spelling_dic = pyphen.Pyphen(lang='en')
+
+URL = "http://127.0.0.1:5000"
+LANGUAGE = "english"
+
+
+print(ipa_dic.inserted("sʌbdʒɛktɪv"))  # use dark magic space trick in the future
+
+wordlist = "allprons.txt"
+data = open(wordlist, mode="r", encoding="utf8").read()
+
+separators = data.splitlines()[0].split()
+
+raw_data = data.splitlines()[1:]
+
+#print(raw_data)
+
+stop_chars ="̈ᵊ̰̚˞͡ˀ̪̃ʔʰ˩˥ʲ̟̯̬​ˠ̞ʷʱ̝ˁ̻cǀˑ̥̹ˢˈˈ̺͡"
```

Comment:
could this be in external files ? it can also be in a configuration file in yaml format. This file could contain all the variables and constant you need to run your program. you can then use the pyyaml module to load them in one call in a dictionary. that makes it easier, since you have one place where you define your constants instead of having to find in which file is defined what.

Reply:
Yes, I intend to use conf files, especially when multiple languages will be involved